### PR TITLE
Add ADOPTERS.md

### PR DESCRIPTION
**Pull request by gastaldi**
Based on the conversation in Zulip with @maxandersen, and based on https://github.com/keycloak/keycloak/blob/master/ADOPTERS.md, this PR adds an `ADOPTERS.md` file containing a list of organizations that publicly use Quarkus 

/cc @emmanuelbernard @maxandersen @cescoffier

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+ADOPTERS
+========
+
+PUBLIC REFERENCES
+-----------------
+
+This document lists the organization names that use Quarkus based on public information available in blog posts and videos.
```
Comment (gwenneg) [inline ADOPTERS.md]:
```suggestion
This document lists the organizations that use Quarkus based on public information available in blog posts and videos. 
```

Comment (gwenneg) [reply ADOPTERS.md]: This change feels more natural to me, but I'm not an English native speaker so maybe the original version is better.

File: ADOPTERS.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+ADOPTERS
+========
+
+PUBLIC REFERENCES
+-----------------
+
+This document lists the organizations that use Quarkus based on public information available in blog posts and videos. 
+If organizations would like get added or removed please make a PR and follow these rules to comply with legal:
```
Comment (Sanne) [inline ADOPTERS.md]: I'd rephrase slightly:

```
If any organization would like get added or removed please make a pull request, and follow these guidelines:
```

(changed organizations from plural form, and omitting "comply with legal" which sounds scary - it's just guidelines anyway)

File: ADOPTERS.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+ADOPTERS
+========
+
+PUBLIC REFERENCES
+-----------------
+
+This document lists the organizations that use Quarkus based on public information available in blog posts and videos. 
+If organizations would like get added or removed please make a PR and follow these rules to comply with legal: 
+
+* Do not include the logo
```
Comment (Sanne) [inline ADOPTERS.md]: would be best to clarify which logo, e.g. not referring to the Quarkus one.

```
Please don't include your organization's logo or other trademarked material
```

File: ADOPTERS.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+* Add a reference (link to a public blog post, video, slides, etc) mentioning that Quarkus is used
```
Comment (Sanne) [inline ADOPTERS.md]: What if there is no public blog post/video/etc..? I think it should be clear it's optional: we do like to have blogposts, but let's not make the barrier too high?

**Comment by Sanne** [comment]
Many thanks for starting this! Highly appreciate it - yet added some nitpicking, I hope you don't mind, best to get this worded at its best :)

File: ADOPTERS.md (hunk 4 above)
Comment (gastaldi) [reply ADOPTERS.md]: I'm afraid that legal might not like having no factual/public proof about Quarkus, that's why I made it required.

**Comment by gastaldi** [comment]
Thank you @Sanne , this is exactly the kind of feedback I was expecting. Feel free to nitpick all the way :wink

File: ADOPTERS.md (hunk 4 above)
Comment (Sanne) [reply ADOPTERS.md]: If it's people from organization X claiming they use it, there's no reason to not accept such a pull request - it's their patch after all, so their own word.

Comment (maxandersen) [reply ADOPTERS.md]: the PR to add it or zulip chats with the claims should be good too.

Comment (gastaldi) [reply ADOPTERS.md]: Any suggestions on how to reword that properly?

**Comment by maxandersen** [comment]
I think this is a nice and simple approach; lets take it on team call tomorrow and see if any objections.

**Comment by cescoffier** [comment]
@emmanuelbernard I let you have the final decision about this one.

**Review by Sanne — APPROVED** [review]

**Comment by maxandersen** [comment]
was agreed yesterday to put it in.

**Comment by machi1990** [comment]
Cool. Maybe  we can add a noteworthy label to this? @gastaldi @maxandersen WDYT?
